buildbot.mariadb.org/ci_build_images/fedora.Dockerfile

```diff
@@ -43,12 +43,14 @@ RUN dnf -y upgrade \
     unixODBC \
     unixODBC-devel \
     wget \
-    && if [ "$(uname -m)" = "x86_64" ]; then dnf -y install libpmem-devel; fi \
+    && arch=$(uname -m) \
+    && if [ "$arch" = x86_64 ] || [ "$arch" = ppc64le ]; then dnf -y install libpmem-devel; fi \
     && dnf clean all \
-    && case $(uname -m) in \
+    && case $arch in \
         "x86_64") curl -sL "https://github.com/tianon/gosu/releases/download/1.14/gosu-amd64" >/usr/local/bin/gosu ;; \
         "aarch64") curl -sL "https://github.com/tianon/gosu/releases/download/1.14/gosu-arm64" >/usr/local/bin/gosu ;; \
         "ppc64le") curl -sL "https://github.com/tianon/gosu/releases/download/1.14/gosu-ppc64el" >/usr/local/bin/gosu ;; \
+        *) curl -sL "https://github.com/tianon/gosu/releases/download/1.14/gosu-$arch" >/usr/local/bin/gosu ;; \
```

Comment:
I am not sure about that, which arch do you have in mind?
See also: https://github.com/tianon/gosu/issues/92

Reply:
nothing specific, it was just obvious to add. And will show up failure if `$arch` isn't right.

Reply:
ok then I suggest we exit with a clear message because as it is now, it can never match how gosu folks name their binaries.